pkg/inframetadata/payload/payload.go

```diff
@@ -0,0 +1,67 @@
+package payload
+
+import (
+	"github.com/DataDog/opentelemetry-mapping-go/pkg/inframetadata/gohai"
+)
+
+// HostMetadata includes metadata about the host tags,
+// host aliases and identifies the host as an OpenTelemetry host
+type HostMetadata struct {
+	// Meta includes metadata about the host.
+	Meta *Meta `json:"meta"`
+
+	// InternalHostname is the canonical hostname
```

Comment:
What is the difference between this hostname and the meta hostname?

Reply:
Addressed on #105